vars/main.yml

```diff
@@ -11,8 +11,13 @@ _artifactory_requirements:
 
 artifactory_requirements: "{{ _artifactory_requirements[ansible_os_family] | default(_artifactory_requirements['default']) }}"
 
-artifactory_zip: "https://dl.bintray.com/jfrog/artifactory/jfrog-artifactory-oss-{{ artifactory_version }}.zip"
-artifactory_home: "{{ artifactory_download_directory }}/artifactory-oss-{{ artifactory_version }}"
+# Pick the Artifactory flavour to install, can be also cpp-ce, jcr, pro.
```

Comment:
The `vars/main.yml` file is for variables used by the role, but are not "exposed" to the user. The `defaults/main.yml` file is intended for users to overwrite values.

Can you move the `artifactory_flavour` variable (and the nice description) to `defaults/main.yml`?

Reply:
Sure, will do.

Additional question though: sice the `artifactory_zip` might need to be overridden, shall I move it as well or leave it as-is?

Reply:
I think the `artifactory_zip` is a variable that the role can determine, based on variable set in `defaults/main.yml`, so it's in the right place in my opinion.

Reply:
Sounds good.
Changes done.